Update Meeting4.md (#124)
atomone-hub/genesis: GovGen_Townhalls/Meeting4.md

- Validator Onboarding Process & Timeline

>   - Tuesday, 20th Feb- publish the Genesis - link will be up in a link in the README of https://github.com/atomone-hub/govgen-genesis

Reviewer: ```suggestion
  - Tuesday, 20th Feb- publish the Genesis - link is up in the [README.md](https://github.com/atomone-hub/govgen-genesis/blob/main/README.md#genesis-download-link) of https://github.com/atomone-hub/govgen-genesis
```

Reviewer: Noticing also my horrible english here, it slipped through as we were rushing last minute changes

>   - Friday 29th, 5PM PST- validators submit your Gentx cut off time

Reviewer: ```suggestion
  - Friday 23th, 5PM PST- validators submit your Gentx cut off time
```
